project-base/storefront/components/Pages/Customer/Orders/OrderPaymentStatusBar.tsx

```diff
@@ -6,16 +6,45 @@ import { twMergeCustom } from 'utils/twMerge';
 type OrderPaymentStatusBarProps = {
     orderPaymentType: string;
     orderIsPaid: boolean;
-    orderIsPaymentInProcess: boolean;
+    orderHasPaymentInProcess: boolean;
+};
+
+const OrderPaymentStatus: FC<{
+    orderIsPaid: boolean;
+    orderHasPaymentInProcess: boolean;
+}> = ({ orderIsPaid, orderHasPaymentInProcess }) => {
+    const { t } = useTranslation();
+
+    if (orderIsPaid) {
+        return (
+            <>
+                <InfoIconInCircle className="w-4 text-backgroundSuccessMore" />
```

Comment:
Just to be safe, I would set the icon `size-4` instead of just the width. Please, check other icons as well.

Reply:
fixed